.circleci/config.yml

```diff
@@ -19,4 +19,4 @@ jobs:
           command: bundle exec jekyll build
       - run:
           name: Test with htmlproofer
-          command: bundle exec htmlproofer ./_site --check-html --allow-hash-href --empty-alt-ignore --disable-external
+          command: bundle exec htmlproofer ./_site --checks --allow-hash-href --ignore-empty-alt --disable-external
```

Comment:
Can we double-check this test? I don't think it's running anymore because it finishes in under a second and says:

```bash
Running 0 checks () in ["./_site"] on *.html files...

Checking 0 external links
Checking 0 internal links
Ran on 232 files!
```

It previously ran for about 6 seconds and said:

```bash
bundle exec htmlproofer ./_site --check-html --allow-hash-href --empty-alt-ignore --disable-external

Running ["ScriptCheck", "LinkCheck", "ImageCheck", "HtmlCheck"] on ["./_site"] on *.html... 

Ran on 232 files!

HTML-Proofer finished successfully.
```

See build [here [circleci]](https://app.circleci.com/pipelines/github/GSA/plainlanguage.gov/328/workflows/0519b0ea-2ae4-461c-9233-ff8482194b73/jobs/1305):

Reply:
@mejiaj Thanks for catching this, I've updated the flags with the new API syntax.  See [here for the new options](https://github.com/gjtorikian/html-proofer/blob/main/bin/htmlproofer).

I've added the flag `--enforce-https=false` to not check jekyll generated links during the build process. Let me know if we should turn this back on?